domain/build.gradle

```diff
@@ -4,12 +4,6 @@ plugins {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
-}
-dependencies {
-
-    api 'org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.7.0'
-    // Coroutines
-    api 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4'
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
```

Comment:
Besides this section, we're using `JavaVersion.VERSION_11` everywhere else. Can we improve our code consistency?

Reply:
yeah, sure will make changes.